processing/src/main/java/io/druid/segment/incremental/IncrementalIndexSchema.java

```diff
@@ -67,19 +70,26 @@ public DimensionsSpec getDimensionsSpec()
     return metrics;
   }
 
+  public boolean isRollup()
+  {
+    return rollup;
+  }
+
   public static class Builder
   {
     private long minTimestamp;
     private QueryGranularity gran;
     private DimensionsSpec dimensionsSpec;
     private AggregatorFactory[] metrics;
+    private boolean rollup;
 
     public Builder()
     {
       this.minTimestamp = 0L;
       this.gran = QueryGranularities.NONE;
       this.dimensionsSpec = new DimensionsSpec(null, null, null);
       this.metrics = new AggregatorFactory[]{};
+      this.rollup = true;
```

Comment:
minor nit : would be nice If we can extract it as a constant somewhere, DEFAULT_ROLLUP, will be helpful if we plan to change the default in future.